temboard-agent.service

```diff
@@ -0,0 +1,27 @@
+# It's not recommended to modify this file in-place, because it will be
+# overwritten during package upgrades.  If you want to customize, the
+# best way is to create a file "/etc/systemd/system/temboard-agent.service",
+# containing
+#	.include /lib/systemd/system/temboard-agent.service
+#	...make your changes here...
+# For more info about custom unit files, see
+# http://fedoraproject.org/wiki/Systemd#How_do_I_customize_a_unit_file.2F_add_a_custom_unit_file.3F
+
+[Unit]
+Description=temBoard agent for PostgreSQL
+After=network.target syslog.target
+
+[Service]
+Type=forking
+
+User=postgres
+Group=postgres
+
+PIDFile=/var/run/postgresql/temboard-agent.pid
```

Comment:
I think we should not daemonize and manage pidfile when running in systemd. This is legacy :)

http://www.mikeperham.com/2014/09/22/dont-daemonize-your-daemons/
http://jdebp.eu/FGA/systemd-house-of-horror/daemonize.html